pkg/storage/unified/sql/data/resource_stats.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+  {{ .Ident "namespace" }},
+  {{ .Ident "group"     }},
+  {{ .Ident "resource"  }},
+  COUNT(*),
+  MAX({{ .Ident "resource_version" }})
```

Comment:
Isn't there only one row per ns+group+resource in the `resource` table (the latest version)? If so, are the count and max needed here?